integrations-and-supported-tools/transformers/notebooks/Neptune_Transformers.ipynb

```diff
@@ -0,0 +1,382 @@
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Before you start\n",
+    "\n",
+    "This notebook example lets you try out Neptune anonymously, with zero setup.\n",
+    "\n",
+    "* If you're running the notebook on your local machine, you need to have [Python](https://www.python.org/downloads/) and [pip](https://pypi.org/project/pip/) installed.\n",
+    "* If you want to see the example logged to your own workspace instead:\n",
+    "    * Create a Neptune account → [Take me to registration](https://neptune.ai/register)\n",
+    "    * Create a Neptune project that you will use for tracking metadata → [Tell me more about projects](https://docs.neptune.ai/administration/projects)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install Neptune and dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install neptune-client transformers==4.22.0 datasets torch==1.12 scipy scikit-learn"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "## Start a run\n",
+    "\n",
+    "To connect your script to Neptune and create a new run, we tell Neptune:\n",
+    "* **Who you are** - with a Neptune API token\n",
+    "* **Where to send your data** - to a Neptune project\n",
+    "\n",
+    "The cell below lets you record data to the public project [common/quickstarts](https://app.neptune.ai/common/quickstarts) as an anonymous user."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import neptune.new as neptune\n",
+    "\n",
+    "run = neptune.init_run(\n",
+    "    api_token=neptune.ANONYMOUS_API_TOKEN,\n",
+    "    project=\"common/quickstarts\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "Alternatively, you can log the example to your own workspace.\n",
+    "\n",
+    "To do that, replace the code above with the following:\n",
+    "\n",
+    "```python\n",
+    "from getpass import getpass\n",
+    "my_api_token = getpass('Enter your Neptune API token: ')\n",
+    "\n",
+    "run = neptune.init_run(\n",
+    "    api_token=my_api_token,\n",
+    "    project=\"workspace-name/project-name\",  # replace with your own\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "For example, if your workspace name is `ml-team` and the project name is `classification`, the project argument is: `project=\"ml-team/classification\"`\n",
+    "\n",
+    "To find your API token and project name, [log in to Neptune](https://app.neptune.ai/).\n",
```

Comment:
@normandy7 - Is this how we are communicating getting the API token and project name? The [API tour notebook](https://colab.research.google.com/github/neptune-ai/examples/blob/master/how-to-guides/how-it-works/notebooks/Neptune_API_Tour.ipynb#scrollTo=XuP4os0AcpuJ) communicates it as below
![image](https://user-images.githubusercontent.com/41324509/191659355-c580d67c-ecf7-4d34-be4e-5002ea8162e3.png)